org.deidentifier.arx.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: org.deidentifier.arx
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+command: arx
+finish-args:
+  - --socket=x11
+#  - --device=dri
+  - --share=ipc
+  # network: the app can connect to DBMS
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk17/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: arx-binary
+    buildsystem: simple
+    build-commands:
+        - install -D -ma+rx arx.sh /app/bin/arx
+        - install -D -ma+r arx-gtk-64.jar /app/bin/arx-gtk-64.jar
+        - install -D -ma+r org.deidentifier.arx.desktop /app/share/applications/org.deidentifier.arx.desktop
+        - install -D -ma+r org.deidentifier.arx.appdata.xml /app/share/metainfo/org.deidentifier.arx.appdata.xml
+        - install -D -ma+r org.deidentifier.arx.svg /app/share/icons/hicolor/scalable/apps/org.deidentifier.arx.svg
+        - install -D -ma+r org.deidentifier.arx.png /app/share/icons/hicolor/128x128/apps/org.deidentifier.arx.png
+
+    sources:
+      - type: file
+        path: arx.sh
+
+      - type: file
+        path: org.deidentifier.arx.appdata.xml
+
+      - type: file
+        path: org.deidentifier.arx.desktop
+
+      - type: file
+        path: org.deidentifier.arx.png
+
+      - type: file
+        path: org.deidentifier.arx.svg
+
+      - type: file
+        url: https://github.com/arx-deidentifier/arx/releases/download/v3.9.0/arx-3.9.0-gtk-64.jar
+        sha256: 2e5527de2dcdebadc31f72bc6cbec385aa1cea6aa9ed65fd73193b325ac99a5c
+        dest-filename: arx-gtk-64.jar
```

Comment:
`gtk-64` does that mean will only work on x86_64 despite being in Java?

Reply:
that's a question for @prasser. I don't have any non x86_64 hardware to test.
Once we can build from source, it should also run on other platforms.

Reply:
The JAR contains some .so files, but I don't know if they are actually required or just for some Optional feature

Reply:
because currently there is another 64-bits architecture built for this flatpak, but I doubt the "64" here mean "any 64-bit architecture".

Reply:
Yes, x86_64 only.

Reply:
that's disappointing.

Reply:
ARX can be built for PPCLE as well, but we currently don't do that. Would that be needed for ARX to be added?

Reply:
aarch64?

I advocate building from source in general, but because it's java, I just don't bother.